services/cypress/cypress.config.ts

```diff
@@ -15,6 +15,7 @@ module.exports = defineConfig({
         videosFolder: 'videos',
         viewportHeight: 1080,
         viewportWidth: 1440,
+        experimentalRunAllSpecs: true,
```

Comment:
This doesn't seem to be working for me. When I run all tests, only `documents.spec.ts` shows up, but never runs.

Reply:
@JasonLin0991 Yes -  the overall test runs are broken (its still in experimental stage, I think that's why I was reading github issues a bit). However, with this flag you can now run by individual folder (so just the cms workflow tests or just the state workflow tests). I thought that was still an incremental improvement from behavior with the flag. Thoughts?

Reply:
Sounds good to me!